modules/qna/src/views/full/utils/qnaList.utils.ts

```diff
@@ -222,16 +223,6 @@ export const fetchReducer = (state: State, action): State => {
       ...state,
       expandedItems: {}
     }
-  } else if (action.type === 'disableQnA') {
-    const { index } = action.data
-    const newItems = state.items
-
-    newItems[index].enabled = false
-
-    return {
-      ...state,
-      items: newItems
-    }
```

Comment:
Why removing the disable method?

Reply:
It wasn't used, it disables through the normal update method